src/pages/AssetPage/assetPage.test.tsx

```diff
@@ -4,46 +4,99 @@ import { render, screen, waitForElementToBeRemoved } from '@testing-library/reac
 import { getUserAssets } from '../../api/assets-api';
 import { useWalletHook } from '../../components/NavBar/useWallethook';
 
-const assetData = {
-    result: [{
-        image_url: 'img',
-        collection: {
-            name: 'colllectionName'
-        },
-        name: 'testname',
-    }]
-}
-
-const wallet = {
-    walletInfo: {
-        address: "0xe3ece548F1DD4B1536Eb6eE188fE35350bc1aa1a"
-    }
-}
-
 jest.mock('../../api/assets-api');
 jest.mock('../../components/NavBar/useWallethook')
 
 describe('Asset page', () => {
-    beforeEach(() => {
-        (useWalletHook as jest.Mock).mockReturnValue(wallet);
-        (getUserAssets as jest.Mock).mockResolvedValue(assetData);
-    });
+    describe('when the wallet information exists', () => {
+        const wallet = {
+            walletInfo: {
+                address: "0xe3ece548F1DD4B1536Eb6eE188fE35350bc1aa1a"
+            }
+        }
+
+        beforeEach(() => {
+            (useWalletHook as jest.Mock).mockReturnValue(wallet);
+        })
+
+        test('renders cards if request succeeds', async () => {
+            const assetData = {
```

Comment:
How about renaming to `oneItemAssetData` and moving it outside because I see this variable is used in several places?